### PR TITLE
AG-12120 Always show data point for financial chart status bar

### PR DESCRIPTION
**Pull request by jacobp100**
https://ag-grid.atlassian.net/browse/AG-12120

### CODE REVIEW

File: packages/ag-charts-enterprise/src/features/status-bar/statusBar.ts [hunk 1]
```diff
@@ -387,23 +387,26 @@ export class StatusBar
...
-        if (activeHighlight == null) {
+        if (datum == null) {
             this.labelGroup.visible = false;
             return;
         }
 
         this.labelGroup.visible = true;
 
-        const datum = activeHighlight.datum;
+        const itemId = activeHighlight?.itemId;
```
Comment (lsjroberts) [inline packages/ag-charts-enterprise/src/features/status-bar/statusBar.ts:407]: Just double checking, should this be `datum?.itemId`?

**Review by lsjroberts — APPROVED** [review]
LGTM, one question:

File: packages/ag-charts-enterprise/src/features/status-bar/statusBar.ts (hunk 1 above)
Comment (jacobp100) [reply packages/ag-charts-enterprise/src/features/status-bar/statusBar.ts:407]: https://github.com/ag-grid/ag-charts/pull/2372/files#diff-f8840d9e039fb128d737560b5bcbbe56bfeda18d24bea54bb311c178088410bdL406

It's refactoring this line

**Pull request merged**